doc/introduction/index.rst

```diff
@@ -103,10 +103,10 @@ Experience (IVE) solutions.
 As a reference implementation, ACRN provides the basis for embedded
 hypervisor vendors to build solutions with a reference I/O mediation
 solution.  In this scenario, an automotive SDC system consists of the
-Instrument Cluster (IC) system in VM1, the In-Vehicle Infotainment (IVI)
-system in VM2, and one or more Rear Seat Entertainment (RSE) systems in
-VM3. Each system is running as an isolated Virtual Machine (VM) for
-overall system safety considerations.
+Instrument Cluster (IC) system running in the Service VM, the In-Vehicle
```

Comment:
change to ``the Service VM and the...``

Reply:
Also change the ``:width: 400px`` to be ``:width: 600px`` in the referenced image (line 97)